packages/strapi-admin/models/User.settings.json

```diff
@@ -1,16 +1,30 @@
 {
   "collectionName": "strapi_administrator",
```

Comment:
While I know this one is a tad older coming from the beta, I'm wondering if we should have an RFC to describe how plugins and packages name their collections by default. With the `users-permissions` you can clearly identify which plugin the collection/table belongs to.

Reply:
That's an init PR. We will certainly change the names later during the dev

Reply:
Alright cool, it just seemed odd to start the collection name with `strapi_` (not sure why this was chosen for `strapi-admin` anyway)